app/models/katello/module_stream.rb

```diff
@@ -14,7 +14,6 @@ class ModuleStream < ApplicationRecord
     scoped_search on: :context, complete_value: true
     scoped_search on: :arch, complete_value: true
     scoped_search relation: :repositories, on: :name, rename: :repository, complete_value: true
-    scoped_search relation: :repositories, on: :id, rename: :repository_id, complete_value: true
```

Comment:
I'm still seeing two links in the UI using this search
```
engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-repositories.html:            <a ng-href="/module_streams?search=repository_id={{repository.id}}">
engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version-yum.html:            <a ng-href="/module_streams?search=repository_id={{repository.id}}">
```

Reply:
nice catch!. Addressed